src/Admin/Preview.php

```diff
@@ -4,52 +4,13 @@
 
 use GraphQL\Error\UserError;
 use WPGraphQL\Router;
+use GraphQLRelay\Relay;
```

Comment:
Is this being used anywhere?

Reply:
Looks like it's not anymore, good catch!